### PR TITLE
Navbar changes

### PR DESCRIPTION
**Pull request by pranay2401**
# CHANGELOG
* Added logo in navbar
* Made the header sticky
* Updated header color to white

### CODE REVIEW

File: packages/im-components/src/molecules/logo/Logo.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <Link to={"/"} className={styles.logo} >
+        <VideoCameraFilled className={styles.image} style={{ fontSize:"250%" }} />
+        <div className={styles.text} >{text}</div>
+      </Link>
+    </>
+  );
+}
+
+Logo.propTypes = {
+  text: PropTypes.string
```
Comment (ukirderohit) [inline packages/im-components/src/molecules/logo/Logo.js]: add children to props
and styles import should be at bottom after constants

File: packages/im-components/src/molecules/logo/logo.module.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  align-items: center;
+}
+
+.logo:hover {
+  color: #000
+}
+
+.text {
+  font-weight: bolder;
+  font-family: monospace;
```
Comment (ukirderohit) [inline packages/im-components/src/molecules/logo/logo.module.scss]: font-family should be of the entire app, then we can change for each section
are we using mono-space overall? if yes then it should be in app.css

File: packages/im-ott/src/organisms/appSkeleton/appskeleton.module.scss [hunk 3]
```diff
@@ -3,6 +3,12 @@
 
 .header {
   color: #000;
+  background: #fff !important;
```
Comment (ukirderohit) [inline packages/im-ott/src/organisms/appSkeleton/appskeleton.module.scss]: avoid using important

**Review by ukirderohit — CHANGES_REQUESTED** [review]

File: packages/im-components/src/molecules/logo/logo.module.scss (hunk 2 above)
Comment (pranay2401) [reply packages/im-components/src/molecules/logo/logo.module.scss]: It is only for the logo. monospace is because we have appLoader with same font-family.

**Review by ukirderohit — APPROVED** [review]

**Pull request merged**